third_party/pyth/p2w-sdk/js/src/index.ts

```diff
@@ -1,116 +1,141 @@
 import { getSignedVAA, CHAIN_ID_SOLANA } from "@certusone/wormhole-sdk";
 import { zeroPad } from "ethers/lib/utils";
 import { PublicKey } from "@solana/web3.js";
-import { PriceFeed, PriceStatus, UnixTimestamp } from "@pythnetwork/pyth-sdk-js";
-
-let _P2W_WASM: any = undefined;
+import { PriceFeed, Price, UnixTimestamp } from "@pythnetwork/pyth-sdk-js";
 
+let _P2W_WASM: any;
 
 async function importWasm() {
-    if (!_P2W_WASM) {
-	if (typeof window === 'undefined') {
-	  _P2W_WASM = await import("./solana/p2w-core/nodejs/p2w_sdk");
-	} else {
-	  _P2W_WASM = await import("./solana/p2w-core/bundler/p2w_sdk");
-	}
+  if (!_P2W_WASM) {
+    if (typeof window === "undefined") {
+      _P2W_WASM = await import("./solana/p2w-core/nodejs/p2w_sdk");
+    } else {
+      _P2W_WASM = await import("./solana/p2w-core/bundler/p2w_sdk");
     }
-    return _P2W_WASM;
+  }
+  return _P2W_WASM;
 }
 
 export type PriceAttestation = {
-    productId: string;
-    priceId: string;
-    price: string;
-    conf: string;
-    expo: number;
-    emaPrice: string;
-    emaConf: string;
-    status: PriceStatus;
-    numPublishers: number;
-    maxNumPublishers: number;
-    attestationTime: UnixTimestamp;
-    publishTime: UnixTimestamp;
-    prevPublishTime: UnixTimestamp;
-    prevPrice: string;
-    prevConf: string;
+  productId: string;
+  priceId: string;
+  price: string;
+  conf: string;
+  expo: number;
+  emaPrice: string;
+  emaConf: string;
+  status: number;
```

Comment:
why did this change from `PriceStatus` ?

Reply:
Because we don't have price status anymore :D